tests/integration/clientv3/metrics_test.go

```diff
@@ -249,42 +263,26 @@ func TestAllMetricsGenerated(t *testing.T) {
 	// Define the expected list of metrics
 	expectedMetrics := []string{
 		"etcd_cluster_version",
-		"etcd_disk_backend_commit_duration_seconds_bucket",
-		"etcd_disk_backend_commit_duration_seconds_count",
-		"etcd_disk_backend_commit_duration_seconds_sum",
-		"etcd_disk_backend_defrag_duration_seconds_bucket",
-		"etcd_disk_backend_defrag_duration_seconds_count",
-		"etcd_disk_backend_defrag_duration_seconds_sum",
-		"etcd_disk_backend_snapshot_duration_seconds_bucket",
-		"etcd_disk_backend_snapshot_duration_seconds_count",
-		"etcd_disk_backend_snapshot_duration_seconds_sum",
+		"etcd_disk_backend_commit_duration_seconds",
+		"etcd_disk_backend_defrag_duration_seconds",
+		"etcd_disk_backend_snapshot_duration_seconds",
```

Comment:
This looks like a breaking change? It changes the metrics names.

Reply:
nope! the _bucket, _count, _sum suffixes are dropped as i am parsing them correctly using the official parser.

you can see lines 266-268 have the correct names.